Coverage/CoverageRunner.h

```diff
@@ -25,388 +25,388 @@
-						}
-						while (!bFound && *p); // end of string
-					}
-				}
-				bSuccess = TRUE;
-				UnmapViewOfFile(pMem);
-			}
-
-			CloseHandle(hFileMap);
-		}
-
-		return strFilename;
-	}
-
-	void InitializeDebugInfo(HANDLE proc)
-	{
-		BOOL initSuccess = SymInitialize(proc, NULL, FALSE);
-		debugInfoAvailable = (initSuccess == TRUE);
-		if (!debugInfoAvailable)
-		{
-			if (!quiet)
-			{
-				std::cout << "Cannot calculate code coverage: debug info is not available." << std::endl;
-			}
-		}
-	}
-
-	void TryPatchDebuggerPresent(HANDLE proc, LPHANDLE fileHandle, PVOID basePtr, std::string filename)
-	{
-		auto idx = filename.find_last_of('\\');
-		if (idx != std::string::npos)
-		{
-			filename = filename.substr(idx + 1);
-		}
-
-		if (Util::InvariantEquals(filename, "KernelBase.dll"))
-		{
-			if (!quiet)
-			{
-				std::cout << "Patching IsDebuggerPresent() in KernelBase.dll" << std::endl;
-			}
-
-			// Find relative address of IsDebuggerPresent using our own version. We're sure it's the same, so why not:
-			auto module = GetModuleHandle("KernelBase.dll");
-			auto modAddr = reinterpret_cast<BYTE*>(module);
-			auto fcnAddr = GetProcAddress(module, "IsDebuggerPresent");
-
-			PVOID processAddress =
-				reinterpret_cast<PVOID>(
-					reinterpret_cast<BYTE*>(basePtr) +
-					(reinterpret_cast<DWORD64>(fcnAddr) - reinterpret_cast<DWORD64>(modAddr))); // Yuck!
-
-			// Patch the data:
-			// 
-			// Signature of method: 
-			// BOOL __stdcall IsDebuggerPresent() { return false; }
-			BYTE bytes[3] = // return false;
-			{
-				0x33, 0xC0, // xor eax, eax
-				0xC3        // ret
-			};
-
-			// Write back the original data:
-			SIZE_T written;
-			WriteProcessMemory(proc, processAddress, bytes, 3, &written);
-			FlushInstructionCache(proc, processAddress, 3);
-
-			debuggerPresentPatched = true;
-		}
-	}
-
-	void ProcessDebugInfo(ProcessInfo* proc, LPHANDLE fileHandle, PVOID basePtr, const std::string& filename)
-	{
-		if (debugInfoAvailable)
-		{
-			bool firstTimeLoad = false;
-			if (loadedFiles.find(filename) == loadedFiles.end())
-			{
-				loadedFiles.insert(filename);
-				firstTimeLoad = true;
-			}
-
-			DWORD64 dllBase;
-			auto idx = proc->LoadedModules.find(basePtr);
-			if (idx != proc->LoadedModules.end())
-			{
-				dllBase = idx->second;
-			}
-			else
-			{
-				dllBase = SymLoadModuleEx(proc->Handle, fileHandle, filename.c_str(), NULL, reinterpret_cast<DWORD64>(basePtr), 0, NULL, 0);
-				proc->LoadedModules[basePtr] = dllBase;
-			}
-
-			if (dllBase)
-			{
-				IMAGEHLP_MODULE64 ModuleInfo;
-				memset(&ModuleInfo, 0, sizeof(ModuleInfo));
-				ModuleInfo.SizeOfStruct = sizeof(ModuleInfo);
-
-				BOOL info = SymGetModuleInfo64(proc->Handle, dllBase, &ModuleInfo);
-
-				// Only register line numbers the first time. On a second load of the same DLL, we only want to set the breakpoints.
-				CallbackInfo ci(&coverageContext, proc, firstTimeLoad);
-
-				if (info)
-				{
+    CoverageRunner(RuntimeOptions& opts) :
```

Comment:
VS must have done all of this whitespace correction... let me know if you want me to undo it...

Reply:
Yes please undo it, it's very hard to see what's going on here...

Reply:
I also cannot comment on the binary (exe) files... +1.43 MB seems excessive - is it a release / debug build? Also x86 seems to be missing an update...

Reply:
Updated the binaries; sizes are pretty close when compared to master's.

Reply:
Whitespace issue should also be resolved now (though I will probably submit a PR just to fix that file at some point since it stands out among the rest).